docs/components/table.md

```diff
@@ -195,6 +195,125 @@ export default {
 </style>
 ```
 
+## Cell Attributes
+
+Add custom properties to individual table cells or groups of cells. The cell attributes object is passed as a param.
+
+### `cellAttrs` Description
```

Comment:
suggestion: The heading style looks nice but the TOC entries don't, do we want to keep it like this?

![image](https://user-images.githubusercontent.com/67973710/117066966-68cb8500-acf7-11eb-8fb0-c08d04f5d97e.png)

I'm thinking follow this style for the Function Description section, we don't really need a TOC entry for it.

![image](https://user-images.githubusercontent.com/67973710/117067668-4e45db80-acf8-11eb-8352-93b2f5704102.png)

For the parameters, maybe we just make the heading `Parameters`? Or at the lease `cellAttrs Parameters` and remove `Function` so it's a bit less wordy.

What do you think?

Reply:
Totally agree, I've made these changes in the latest commit.

Reply:
**New README appearance**
![Screen Shot 2021-05-04 at 2 37 49 PM](https://user-images.githubusercontent.com/2080476/117073600-01a5d480-ace7-11eb-9967-f8612d9d7d61.png)

**TOC**
![Screen Shot 2021-05-04 at 2 38 07 PM](https://user-images.githubusercontent.com/2080476/117073605-023e6b00-ace7-11eb-9d81-28fd7703bc29.png)

Reply:
Looking good!